appinventor/components/src/com/google/appinventor/components/runtime/TextBoxBase.java

```diff
@@ -138,6 +138,7 @@ public TextBoxBase(ComponentContainer container, EditText textview) {
     }
 
     Text("");
+    lastText = "";
```

Comment:
I think it might be better to just set this at the field declaration.

Reply:
Updated.